new/LIT6702MashafaKobor.xml

```diff
@@ -0,0 +1,72 @@
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6702MashafaKobor" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መጽሐፈ፡ ኮቦር፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Maṣḥafa kobor</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A short text predicting different events, mostly of unfortunate character, that will ensue depending on the day of the week on which
+                    the feast day of <persName ref="PRS5741Johnthe"/> falls. This work is attested independently as additional content, and as part of 
+                    <ref type="work" corresp="LIT5048Hassabat"/>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2022-08-09">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="ecrm:CLP46i_may_form_part_of" active="LIT6702MashafaKobor" passive="LIT5048Hassabat"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>The following incipit and explicit are taken from <ref type="mss" corresp="BDLaethe32"/>.</note>
+                <div type="textpart" subtype="incipit">
+                    <ab>ለእመ፡ ኮነ፡ ኮቦር፡ ዮሐንስ፡ በሰኑይ፡ አልቦ፡ ደዌ፡ ለጸብእ፡ አገሳ፡ የሐልቁ፡ ዝናም፡ የሐፅፅ፡ መዓር፡ ይውኅዝ፡
+                        ሰገም፡ ይሰምር፡ ሰናይ፡ ይሰምር፡ ሕፃናተ፡ ይሙውቱ ፨</ab>
```

Comment:
The handwriting is not very clear, but I read the nominative form ሕፃናት (which agrees with the following verb ይመውቱ፡). The vowel marker of ት is just hinted.